src/components/Button/Button.tsx

```diff
@@ -0,0 +1,37 @@
+import { ReactNode } from 'react';
+import { BaseStyleProp } from '../types';
+import { makeStyles } from '../../utils/makeStyles';
+
+type ButtonStyles = 'root';
+
+export interface ButtonProps extends BaseStyleProp<ButtonStyles> {
+  children?: ReactNode;
+  variant?: 'primary' | 'secondary';
+  href: string;
+}
+
+const useStyles = makeStyles({
+  root: {
+    fontSize: '18px',
+    textDecoration: 'none',
+    padding: '10px 16px',
+    borderRadius: '5px',
+    cursor: 'pointer',
+  },
+});
+
+export const Button = ({
+  children,
+  // variant = 'primary',
+  href,
+  classes,
+  className,
+}: ButtonProps): JSX.Element => {
+  const styles = useStyles({ classes });
+
+  return (
+    <a href={href} target="_blank" style={styles.root} className={className}>
```

Comment:
Let's expose a `target` prop and use `"_blank"` as the default value

Reply:
Email links always pop up in a new tab, do you think we need it?

Reply:
Added  `target?: HTMLAttributeAnchorTarget;`  prop and set the default value to  `_blank`